components/ide-service-api/typescript-grpcweb/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "@gitpod/ide-service-api-grpcweb",
+  "version": "0.0.1",
+  "license": "UNLICENSED",
+  "scripts": {
+    "build": "bash build.sh"
+  },
+  "files": [
+    "lib"
+  ],
+  "dependencies": {
+    "grpc-web": "^1.3.1"
```

Comment:
we should use https://github.com/improbable-eng/grpc-web as we do for other services, check out on main supervisor-api and ide-metrics-api

grpc-web does not support Node.js and server is Node.js

Reply:
I'm not sure. Server is using supervisor via grpcweb https://github.com/gitpod-io/gitpod/blob/19455a4a9a31a55d033fe95331cdfe9eacaeec8d/components/server/package.json#L38 How will we use?

Reply:
ok, then let's drop all unnecessary packages